Anlab.Mvc/Views/Admin/MailQueue.cshtml

```diff
@@ -0,0 +1,132 @@
+﻿@model List<Anlab.Core.Domain.MailMessage>
+@{
+    ViewData["Title"] = "Mail Queue";
+}
+
+<div id="collapsebutton" class="btn btn-default"><i class="glyphicon glyphicon-minus"></i> Collapse All</div>
+<div id="expandbutton" class="btn btn-default"><i class="glyphicon glyphicon-plus"></i> Expand All</div>
+
+<div style="padding-bottom:120px;">
+<table id="table" cellpadding="0" class="row-border stripe order-column">
+    <thead>
+        <tr>
+            <th>Status:</th>
+            <th>To:</th>
+            <th>Subject:</th>
+            <th>Sent At:</th>
+        </tr>
+    </thead>
+    <tbody>
+    @foreach (var message in Model)
+    {
+        <tr>
```

Comment:
@laholstege no reason to change but in the future you can actually just create a variable for something like this.  `var sentStatus = message.Sent.HasValue ? "Sent" : "Failed"` and then in the actual <td> you can just do `@sentStatus`.  Have to put in an extra check but the idea is to get the logic into c# instead of doing a bunch of <p> tags.  Just an FYI you can keep this as-is